src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-bar"
+      data-qa="big"
+      action="#"
+      method="get"
+    >
+      <div>
+        <img
+        class="search-bar__loupe"
+        alt="search-loupe"
+        src="./images/Search.svg"
+        >
+        <input
+          class="search-bar__input"
+          placeholder="Try &quot;Los Angeles&quot;"
+          type="text"
+          data-qa="keypress"
+        >
+      </div>
+    </form>
+    <form
```

Comment:
Remember to add spaces between sibling elements for readability
```suggestion
    </form>
    
    <form
```